.claude/prompts/maintenance-audit.md

```
@@ -0,0 +1,203 @@
+
+Check that dependencies are current and clean.
+
+**What to check:**
+- `go mod tidy` — run and check if it produces a diff
+- Outdated dependencies — `go list -m -u all` to find available updates
+- Unused direct dependencies — dependencies in `require` not imported anywhere
+- Known vulnerabilities — `govulncheck ./...` if available
+
+**How to run:**
+- `go mod tidy` dry run: `cp go.sum go.sum.bak && go mod tidy && diff go.sum go.sum.bak; mv go.sum.bak go.sum` (restore after)
+- `go list -m -u all 2>/dev/null | grep '\[' ` — shows deps with available updates
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)